Formula/dockerize.rb

```diff
@@ -0,0 +1,23 @@
+class Dockerize < Formula
+  desc "Utility to simplify running applications in docker containers"
+  homepage "https://github.com/jwilder/dockerize"
+  url "https://github.com/jwilder/dockerize.git",
+      :tag      => "v0.6.1",
+      :revision => "7c5cd7c34dcf1c81f6b4db132ebceabdaae17153"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/jwilder/dockerize").install buildpath.children
+    ENV.append_path "PATH", buildpath/"bin"
```

Comment:
I don't think this is needed

Reply:
That's needed for the Makefile. It installs `glock` via `go get` and then tries to execute it.

As @mrmachine doesn't respond anymore, I created a new PR to address this: https://github.com/Homebrew/homebrew-core/pull/37407.

Please close this PR.